### DISCUSSION TITLE
Informazione campi R009 Matrimonio UC 311111

### DISCUSSION THREAD

**Discussion by eschoysman** [italia/ansc#857] [Supporto servizi cooperativi] [answered]
Buongiorno,

nello sviluppo del caso d'uso del matrimonio in oggetto (311111) sono presenti i campi tipoAssistenteSposo e tipoAssistenteSposa.

A quale tabella di decodifica corrispondono? Nei documenti csv messi a disposizione (https://github.com/italia/ansc/blob/main/docs/Mapping_casi_uso/matrimoni/Matr_001.csv), i campi possono assumere valori da 1 a 5, ma non trovo la decodifica relativa.

**Comment by mttfranci** [comment, marked as answer]
Salve, come indicato nelle specifiche [OpenAPI](https://github.com/italia/ansc/blob/v1.29.5/docs/openapi/model_evento.yaml#L2361) i valori si possono trovare nella tabella [51](https://github.com/italia/ansc/blob/main/docs/Decodifiche/51_dec_assistenza_minori.csv),

Saluti

Assistenza ANSC
